Wikipedia/Networking/Fetchers/SavedArticlesFetcher.h

```diff
@@ -17,14 +23,14 @@
 + (SavedArticlesFetcher*)sharedInstance;
 + (void)                 setSharedInstance:(SavedArticlesFetcher*)fetcher;
 
-@property (nonatomic, strong, readonly) MWKSavedPageList* savedPageList;
-@property (nonatomic, strong, readonly) MWKDataStore* dataStore;
-
-- (instancetype)initWithDataStore:(MWKDataStore*)dataStore;
+@property (nonatomic, strong) MWKSavedPageList* savedPageList;
 
 @property (nonatomic, weak) id<SavedArticlesFetcherDelegate> fetchFinishedDelegate;
 
-- (void)fetchSavedPageList:(MWKSavedPageList*)savedPageList;
+- (instancetype)initWithDataStore:(MWKDataStore*)dataStore;
```

Comment:
This, along with your other comments about setter methods and other things, are tied to the root cause of me not wanting to break _too_ much of the existing API, and thereby also having to fix or delete the legacy `SavedPagesViewController`. If we're all on the same page that it's going to be deleted eventually, I can make more drastic changes to the API which assume:

- The saved page list never changes*
- There's no `sharedInstance` of this class

\* This will probably be true for quite some time, but I hope we'll eventually do something crazy like support multiple logins & data stores